backend/dataall/modules/omics/tasks/omics_workflows_fetcher.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import sys
+import datetime
+
+from dataall.core.environment.db.environment_models import Environment
+from dataall.base.db import get_engine
+from dataall.modules.omics.aws.omics_client import OmicsClient
+from dataall.modules.omics.db.omics_models import OmicsWorkflow
+from dataall.modules.omics.api.enums import OmicsWorkflowType
+from dataall.modules.omics.db.omics_repository import OmicsRepository
+
+
+root = logging.getLogger()
+root.setLevel(logging.INFO)
+if not root.hasHandlers():
+    root.addHandler(logging.StreamHandler(sys.stdout))
+log = logging.getLogger(__name__)
+
+
+def fetch_omics_workflows(engine):
+    """List Omics workflows."""
+    log.info('Starting omics workflows fetcher')
+    with engine.scoped_session() as session:
+        environments = session.query(Environment)
+        for env in environments:
```

Comment:
Do we need to iterate over every environment? Are the list of READY2RUN workflows to be different per environment? 

If so - can we remove dups before then iterating on workflows?

Reply:
Note - if we do keep the logic as is, it needs to be for only environments with `omicsEnabled` or in some `try/except` logic 

Otherwise there will be permission errors with pivotRole

Reply:
Taking out iterate over all envs for now; will need later when we introduce private workflows

Reply:
@dlpzx will investigate only omicsEnabled check